bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+	except Exception as e:
+	    click.echo("wrong password, please try again")
```

Comment:
As we mentioned in your older PR, we don't want to assume that any Exception here means that the password was wrong. You will need to catch whatever more specific Exception subclass is raised when the password is wrong.